### ISSUE TITLE
auth issue on remote connection

### ISSUE THREAD

**Issue by xybydy** [Spazzlo/folderclone#90]
I am unable to complete auth process on ssh connection since i am unable to paste the code I get from google.

Could you please convert the auth process like the one rclone does, so we can paste the api verification code on cmd.

**Comment by smartass08** [comment]
Start the AutoRclone auth in screen or tmux,
Copy the link , 
Detach the screen, 
Complete the verification in your different device , 
Copy the `localhost:whatever' link 
Install w3m in same pc where you are doing auth of Autorclone , 
Type the command - `w3m 'localhost:whatever'`
Attach to your screen or tmux and your verification will be done

**Comment by xybydy** [comment]
Thanks for the workaround but this matter actually needs to be fixed rather than wasting that much effort only for an auth.

**Comment by Mz49** [comment]
You are completely right, @xybydy. We will be sure to implement a headless authentication option in an upcoming rewrite.
